src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -3,8 +3,10 @@
 
 using System;
 using System.Collections.Generic;
+using System.Drawing;

```

Comment:
do we need this one?

Reply:
I'm taking a new approach to math... :)  VS inserts these things on me.  Is there a way to turn that off?  May be worth me doing if there is.